### DISCUSSION TITLE
在swagger中调用创建用户接口报错

### DISCUSSION THREAD

**Discussion by NightClown** [hhyo/Archery#1895] [Q&A]
请求参数中  "resource_group": [1,2,3]；我该如何传这个参数呢

Exception Type: TypeError at /api/v1/user/
Exception Value: Direct assignment to the forward side of a many-to-many set is prohibited. Use groups.set() instead.

**Comment by nick2wang** [comment]
该问题已经修复，请更新最新代码 #1776
